### ISSUE TITLE
Support for visually impaired

### ISSUE THREAD

**Issue by skogsbrus** [johan-andersson01/biblion#11]
To solve this issue:
- Review the color scheme
- Add text alternatives to book covers

> 1. Every <img> must have an alt="" attribute
> 2. Describe the information, not the picture
> 3. Active images require descriptive alt text
> 4. Images that contain information require descriptive alt text
> 5. Decorative images should have empty alt text

See https://webdesign.tutsplus.com/articles/accessibility-basics-designing-for-visual-impairment--cms-27634 for more.
